src/components/file/UpdateContentFile.tsx

```diff
@@ -73,7 +73,7 @@ const UpdateContentFile = ({ file, onClosePopover }: UpdateContentFileProps): JS
 		if (!upload.success || !upload.file) {
 			toast({ title: upload.message, status: upload.success ? 'success' : 'error' });
 		} else {
-			const updated = await user.contact.updateFileContent(upload.file);
+			const updated = await user.fullContact.files.updateFileContent(upload.file);
```

Comment:
```suggestion
			const updated = await user.fullContact.files.updateContent(upload.file);
```
same here and everywhere else